proto/osmosis/concentrated-liquidity/query.proto

```diff
@@ -108,6 +108,13 @@ service Query {
                                    "cfmm_pool_id_link_from_concentrated/"
                                    "{concentrated_pool_id}";
   }
+
+  // GetTotalLiquidity returns total liquidity across all cl pools.
+  rpc GetTotalLiquidity(GetTotalLiquidityRequest)
+      returns (GetTotalLiquidityResponse) {
+    option (google.api.http).get =
+        "/osmosis/concentratedliquidity/v1beta1/get_total_liquidity";
+  }
```

Comment:
Should we also expose this in poolmanager that does this for all pools? AFAIR, Ibrar's requirement was related to all pools

Reply:
I added a method to PoolI of GetTotalLiquidity for each pool type, and exposed a total liquidity endpoint on pool manager here: https://github.com/osmosis-labs/osmosis/pull/5670/commits/098858af7126bcca23b4886a82a295d369371667